libretroshare/src/ft/ftserver.cc

```diff
@@ -1872,15 +1975,17 @@ void ftServer::ftReceiveSearchResult(RsTurtleFTSearchResultItem *item)
 			hasCallback = true;
 
 			std::vector<TurtleFileInfoV2> cRes;
-			for( const auto& tfiold : item->result)
-				cRes.push_back(tfiold);
+			std::transform( item->result.begin(), item->result.end(),
```

Comment:
the other version doesn't compile on all platforms (aka Android), because of that I changed it, moreover std::trasform suggest to the compile to use SIMD instructions when available AFAIR

Reply:
It doesn't compile because of C++11? There's nothing fancy there except the for loop that could be written with an explicit iterator

Reply:
No it is not a C++11 issue, It's because of an implicit conversion that the Android compiler doesn't like, so as I had to rewrite it I have done the way the compiler error message suggested to do.

Reply:
come on! This should work:
for(auto& tfiold: item->result)
    cRes.push_back(TurtleFileInfoV2(tfiold));
Implicit conversion is not accepted because the constructor is explicit.